src/core/public/chrome/chrome_service.tsx

```diff
@@ -111,6 +111,9 @@ export class ChromeService {
   private readonly navLinks = new NavLinksService();
   private readonly recentlyAccessed = new RecentlyAccessedService();
   private readonly docTitle = new DocTitleService();
+  private readonly useCaseList$ = new BehaviorSubject<
+    Array<ChromeUseCase & { navLinks: ChromeRegistrationNavLink[] }>
+  >([]);
```

Comment:
could this be a map?

Reply:
Good idea, internally it is more convenient to insert items into a map, and I am thinking make navLinks as a map as well. But I'd like to keep the output of `getUseCase$` as array.